scipy/spatial/transform/rotation.pyx

```diff
@@ -1446,12 +1576,14 @@ def inv(self):
                [ 0.78539816, -0.        , -0.        ]])
 
         """
-        quat = self._quat.copy()
+        cdef np.ndarray quat = np.array(self._quat, copy=True)
         quat[:, -1] *= -1
         if self._single:
             quat = quat[0]
         return self.__class__(quat, normalize=False, copy=False)
```

Comment:
In master `normalize=True` now. Why is it not displayed as a change here?

I would update local master and rebase once again.

Reply:
Updated